src/workloads/replication/startup/README.md

```diff
@@ -0,0 +1,40 @@
+## BACKGROUND
+
+This directory contains some Genny workloads to test the performance of the mongod shutdown/startup process.
+
+## TEST DESCRIPTION
+
+The test is structured with several Genny workloads that are executed in order by a DSI custom test control that restarts the mongod between the workloads, allowing us to measure shutdown/startup performance.
+
+### The test has two phases:
+  
+  **1. Light load:**
+
+    - Workload names preceded with "1_".
```

Comment:
Is there some significance to the "1_" and "2_", or can more mnemonic names be used?

Reply:
what are you suggesting.

Reply:
It is just to enforce order, as the order matters here.